docs/judge/status_codes.md

```diff
@@ -16,10 +16,9 @@ Your program caused a runtime exception to occur. This will only occur for nativ
 
 | Feedback | Description |
 |----------|-------------|
-| `segmentation fault`, `bus error` | Your program was killed by SIGSEGV or SIGBUS. Generally, this means you ran out of memory, but it can also mean that you are accessing arrays out of bounds, in some cases. |
+| `segmentation fault`, `bus error` | Your program was killed by `SIGSEGV` or `SIGBUS`. Generally, this means you ran out of memory, but it can also mean that you are accessing arrays out of bounds, or opening files (which is generally not allowed). |
```

Comment:
This line is only true if you're writing unsafe code that doesn't error-check. An `fopen` will not by itself segfault. I think we should leave this change out.

Reply:
Fixed